coresdk/src/test/unit_tests/unit_test_json.cpp

```diff
@@ -117,7 +117,18 @@ TEST_CASE("json can be created and read", "[json]")
 
         REQUIRE("#00ff00ff" == color_to_string(deserialized_clr));
     }
+    SECTION("json can be created and read with different number types")
+    {
+        json number_types = create_json();
+
+        json_set_number(number_types, "float", static_cast<float>(21.2));
+        json_set_number(number_types, "double", static_cast<double>(30.1));
```

Comment:
You can write floating point literals without a cast here to make it a bit easier to read:

```suggestion
        json_set_number(number_types, "float", 21.2f);
        json_set_number(number_types, "double", 30.1);
```
[There's a more in depth look at the C++ reference docs](https://en.cppreference.com/w/cpp/language/floating_literal.html)